pkg/handlers/system_intake.go

```diff
@@ -87,15 +89,36 @@ func (h SystemIntakeHandler) Handle() http.HandlerFunc {
 			euaID, ok := appcontext.EuaID(r.Context())
 			if !ok {
 				logger.Error("Failed to get EUA ID from context")
-				http.Error(w, "Failed to PUT system intake", http.StatusInternalServerError)
+				http.Error(w, "Failed to PUT system intake", http.StatusUnauthorized)
 				return
 			}
 			intake.EUAUserID = euaID
+			updatedTime := time.Now().UTC()
```

Comment:
This feels more like `services` responsibility, since it's not part of the request. We should avoid using `time` also, since it's not testable: https://github.com/trussworks/Engineering-Playbook/blob/master/developing/languages/GO.md#time

Reply:
ok, will move it to services